modules/deployments-ab-testing-lb.adoc

```diff
@@ -168,6 +168,16 @@ routes/ab-example       Service  ab-example-a 198 (99%)
 routes/ab-example       Service  ab-example-b 2   (1%)
 ----
 
+. To override the default values for the `load-balancing` algorithm, you can adjust the annotation on the route by setting the algorithm to `round-robin`. The default load balancing algorithm for an {product-title}  route is set to `load-balancing`.
```

Comment:
```suggestion
. To override the default values for the `load-balancing` algorithm, adjust the annotation on the route by setting the algorithm to `round-robin`. The default load balancing algorithm for an {product-title}  route is set to `load-balancing`.
```

Reply:
This procedure is also included in the ROSA docs (for example: https://74142--ocpdocs-pr.netlify.app/openshift-rosa/latest/applications/deployments/route-based-deployment-strategies#deployments-ab-testing-lb-cli_route-based-deployment-strategies), which means there's a minor grammatical issue ("an Red Hat OpenShift Service on AWS route"). You might consider conditionalizing this sentence so that it reads, "a {product-title}" for ROSA, and "an {product-title} for OCP/OSD.

Reply:
Not a big deal but I think there's an extra space after {product-title}. Agree with Ben's suggestion about conditionalizing for ROSA, OSD/OCP.

Reply:
for default load-balancing algorithm, please refer to https://docs.openshift.com/container-platform/4.13/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration  and check the first annotation `haproxy.router.openshift.io/balance`
```
Available options are random, source, roundrobin, and leastconn. The default value is source for TLS passthrough routes. For all other routes, the default is random. 
```
The words `...by setting the algorithm to round-robin` and `...is set to load-balancing` are not accurate.